framework/include/problems/EigenProblem.h

```diff
@@ -121,11 +196,23 @@ class EigenProblem : public FEProblemBase
   bool _negative_sign_eigen_kernel;
 
   unsigned int _active_eigen_index;
+
+  bool _auto_initilize_eigen_vector;
+  bool _do_free_power_iteration;
+  bool _output_inverse_eigenvalue;
+
   /// Timers
   PerfID _compute_jacobian_tag_timer;
   PerfID _compute_jacobian_ab_timer;
   PerfID _compute_residual_tag_timer;
   PerfID _compute_residual_ab_timer;
   PerfID _solve_timer;
   PerfID _compute_jacobian_blocks_timer;
+
+  bool _has_normalization;
+  PostprocessorName _normalization;
+  Real _normal_factor;
```

Comment:
What is the difference between these two? It might be helpful to add some doxygen explaining what these individual members do

Reply:
Yes